pkg/tfbridge/replaceErrorProperties.go

```diff
@@ -0,0 +1,48 @@
+package tfbridge
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/hashicorp/go-multierror"
+	"github.com/pkg/errors"
+	"github.com/pulumi/pulumi-terraform-bridge/v3/unstable/metadata"
+)
+
+func replaceSubstring(s, old, new string) string {
+	return strings.ReplaceAll(s, old, new)
+}
+
+func ReplaceSubstrings(s string, replacements map[string]string) string {
```

Comment:
I think all the functions in this module (except the entry point) should be private.